### DISCUSSION TITLE
RA doesn't work.

### DISCUSSION THREAD

**Discussion by thepeppapower** [MaaAssistantArknights/MaaAssistantArknights#10711] [Q&A] [answered]
After starting the task, the MAA opends RA page and nothing happens. I checked the connection, and also MAA perfectly works for IS. I'm on UI Version: v5.7.0-beta.2 Core Version: v5.7.0-beta.2 rn.
 EN version. Squads deleted, start from RA page

**Comment by Alan-Charred** [comment, marked as answer]
> After starting the task, the MAA opends RA page and nothing happens. I checked the connection, and also MAA perfectly works for IS. I'm on UI Version: v5.7.0-beta.2 Core Version: v5.7.0-beta.2 rn.
> 
>  EN version. Squads deleted, start from RA page

Could you please double check the advanced settings of the RA task? If you would like to farm points without a save, you should choose mode 0, which is not the default option.

If the issue remains, please submit an issue with the corresponding log file.


Best,

**Reply by thepeppapower** [reply]
Thanks. That was the issue, should have read properly.
